### PR TITLE
sql,importccl: plumb database descriptor to avoid lookups

### PR DESCRIPTION
**Pull request by ajwerner**
These lookups to qualify a name for an error when making an FK were bad. The thrust of this change is making qualifyFKColErrorWithDB not do any lookups.

Release note: None

### CODE REVIEW

File: pkg/ccl/importccl/read_import_pgdump.go [hunk 1]
```diff
@@ -342,13 +336,30 @@ func createPostgresSequences(
...
+	schemaAndTableName schemaAndTableName, schemaNameToDesc map[string]*schemadesc.Mutable,
+) (catalog.SchemaDescriptor, error) {
+	var schema catalog.SchemaDescriptor
+	switch schemaAndTableName.schema {
+	case "", "public":
+		schema = schemadesc.GetPublicSchema()
+	default:
+		var ok bool
+		if schema, ok = schemaNameToDesc[schemaAndTableName.schema]; !ok {
+			return nil, errors.Newf("schema %s not found in the schemas created from the pgdump",
```
Comment (postamar) [inline pkg/ccl/importccl/read_import_pgdump.go:349]: `s/%s/%q/` ?

**Review by postamar — APPROVED** [review]

File: pkg/ccl/importccl/read_import_pgdump.go (hunk 1 above)
Comment (ajwerner) [reply pkg/ccl/importccl/read_import_pgdump.go]: Done.

**Comment by ajwerner** [comment]
TFTR! 

bors r=postamar

**Comment by ajwerner** [comment]
bors r=postamar

**Pull request merged**